prog/postgres/postgres_server_nexus.rb

```diff
@@ -31,7 +31,8 @@ def self.assemble(resource_id:, timeline_id:, timeline_access:, representative_a
         boot_image: postgres_resource.project.get_ff_postgresql_base_image || "postgres-ubuntu-2204",
         private_subnet_id: private_subnet_id,
         enable_ip4: true,
-        allow_only_ssh: true
+        allow_only_ssh: true,
+        exclude_host_ids: exclude_host_ids
```

Comment:
In the future, should we also exclude the host running `postgres-monitoring`?

Reply:
I think that makes sense. Though the problem is that `postgres-monitoring` DB also has HA, which means it would block 2 hosts for everyone and the hosts its servers are in would change after a failover. Maybe we should pin the monitoring DB to two specific hosts. We can do that after we have more hosts available for PG use. Currently we are running near capacity for PG/VM workloads. Since this is not meant to be blocker, I'll resolve this and return back in the future.